platform/src/common/components/Collocation/Report/MonitorReport/InterCorrelation/index.jsx

```diff
@@ -94,21 +86,21 @@ const InterCorrelationChart = ({
       endDate,
     });
 
-    if (!response.error) {
-      const updatedQuery = {
-        ...router.query,
-        device: newDevice,
-        startDate,
-        endDate,
-      };
+    // if (!response.error) {
+    //   const updatedQuery = {
+    //     ...router.query,
```

Comment:
@Codebmk, do you need this code?

Reply:
Yeap this one's for intercorrelation charts, commented it out because the v2 API is not ready and didn't want things breaking